api/controllers/v1/jobs.ts

```diff
@@ -47,23 +38,26 @@ export const TriggerLocalBuild = async (event: APIGatewayEvent): Promise<APIGate
   }
 };
 
-export const HandleJobs = async (event: SQSEvent): Promise<void> => {
+// TODO: use @types/aws-lambda
+export const HandleJobs = async (event: any = {}): Promise<any> => {
   /**
    * Check the status of the incoming jobs
    * if it is inqueue start a task
    * if it is inprogress call NotifyBuildProgress
    * if it is completed call NotifyBuildSummary
    */
-  const messages = event.Records;
+  const messages: JobQueueMessage[] = event.Records;
```

Comment:
is this correct? seeing message.body in line 53 and JobQueueMessage has no member body